### ISSUE TITLE
It will be more convenient if NO cooling in 3DTHM  provide outputs in K/s or in W/m^3 instead of K per GITM time step.

### ISSUE THREAD

**Issue by AtishnalChand** [GITMCode/GITM#2]
**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

**Smartphone (please complete the following information):**

**Additional context**

**Comment by aaronjridley** [comment]
Yeah, I agree.  I will change the 3DTHM file to output dT/dt for all variables instead of dT.  That will be added to the list of things to do as I clean up the code.

**Comment by aaronjridley** [comment]
Ok, all the heating rates are now in K/s instead of some in K and some in K/s. (This is in a branch that will be merged into develop today.)
